website/src/controllers/api-v3/groups.js

```diff
@@ -152,6 +152,13 @@ api.getGroup = {
     let group = await Group.getGroup({user, groupId: req.params.groupId, populateLeader: true});
     if (!group) throw new NotFound(res.t('groupNotFound'));
 
+    if (!user.contributor.admin) {
+      _.remove(group.chat, function removeChat (chat) {
```

Comment:
@TheHollidayInn here you probably want to convert the group to a plain object with group.toJSON before removing flags information